photoeditor/src/main/java/ja/burhanrashid52/photoeditor/DrawingView.kt

```diff
@@ -54,8 +54,8 @@ class DrawingView @JvmOverloads constructor(
         // apply shape builder parameters
         currentShapeBuilder?.apply {
             paint.strokeWidth = this.shapeSize
-            paint.alpha = this.shapeOpacity
             paint.color = this.shapeColor
+            paint.alpha = this.shapeOpacity
```

Comment:
I am not sure how changing the order in variable assignment solves the issue? Can you please elaborate?

Reply:
This is because the alpha value is also defined in the setColor method of the Paint class, so if the setColor method is called after the setAlpha method is set, the alpha value will be overwritten.

https://cs.android.com/android/platform/superproject/+/master:frameworks/base/graphics/java/android/graphics/Paint.java;l=1094?q=Paint.java%20setAlpha&ss=android

Reply:
Also, the value that comes in the argument of the withShapeColor method is set to an alpha value of 0xFF.
ex )<color name="blue_color_picker">#0066FF</color> =>  #FF0066FF

Reply:
@burhanrashid52 
I'm not very good at English, so I use an automatic translator.
Perhaps there was something unclear or impolite?

Please let me know if there is anything I have not done correctly.